.env.test

```diff
@@ -1,3 +1,4 @@
 REDIRECTION_DOMAINS=trial.domain:production.domain
 ACTIVE_JOB_ENABLED=false
 PROCESSED_DELETED_PER_PAGE=2
+DWP_NOTIFICATION_ALERT_EMAILS=['dan@test.com', 'petr@test.gov.uk']
```

Comment:
I always think its good practice to freeze array constants - otherwise they are not as constant as you think.

Reply:
This is only for a test so I don't really worry about it.